Sources/MapboxCoreNavigation/PassiveNavigationEventDetails.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import CoreLocation
+
+struct PassiveNavigationEventDetails: NavigationEventDetails {
+    let coordinate: CLLocationCoordinate2D?
+    let created = Date()
+    let sessionIdentifier = "session-id" // TODO: create global session
```

Comment:
Should the session identifier be an extensible enumeration?

Reply:
For `sessionIdentifier` field I reused the `SessionState` class that we already use for active guidance, it represents the session identifier with UUID type.